src/sites/link/davinsurance.js

```diff
@@ -0,0 +1,27 @@
+_.register({
+  rule: {
+    host: /^davinsurance\.com$/,
+    query: /r=(.+)/,
+  },
+  async start (m) {
+    const e = m.query[1];
+
+    function base64_decode(w) {
```

Comment:
We have a native function `atob` for base64 decoding.
Please remove this function.